test/filters/browser_tests.filter

```diff
@@ -9,6 +9,13 @@
 # display an additional confirmation infobar for P3A
 -All/PageSpecificSiteDataDialogBrowserTest.*/*
 
+# These tests fail because they expect an additional Manifest V2 deprecation
```

Comment:
we should add similar Brave tests to check to ensure Manifest V2 is not deprecated

Reply:
added